src/scripts/main.js

```diff
@@ -1,3 +1,22 @@
 'use strict';
 
+const table = document.querySelector('table');
+const thead = document.querySelector('thead');
+const tbody = document.querySelector('tbody');
+const firstRow = thead.rows[0];
+
+const newTh = table.appendChild(document.createElement('th'));
```

Comment:
The new 'th' element should be appended to 'thead' instead of 'table'. This will ensure that the new header cell is correctly added to the header row.